src/index.html

```diff
@@ -1,12 +1,375 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="./images/logo.png"
+            alt="Logo image"
+          >
+        </a>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ФОТО
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ВИДЕО
+            </a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="products">
+        <div class="card products__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="Product image"
+          >
+          <h2 class="card__heading">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
+              <div class="stars-item"></div>
```

Comment:
```suggestion
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
              <div class="stars__item"></div>
```
change the names of the block items to fit BEM methodology